app/scripts/framework/service/meta-service.js

```diff
@@ -161,7 +161,10 @@ org.ekstep.services.metaService = new (org.ekstep.services.iService.extend({
 		this.postFromService(this.getBaseURL() + this.getAPISlug() + this.getConfig('configEndPoint', '/data') + this.getConfig('configurationUrl', '/v1/form/read'), data, this.requestHeaders, callback)
 	},
 
-	getVideoLicense: function (data, callback) {
-		this.postFromService(this.getBaseURL() + this.getAPISlug() + this.getConfig('videoLicenseURL', '/asset/v3/license/validate'), data, this.requestHeaders, callback)
+	getVideoLicense: function (data, field,callback) {
```

Comment:
indentation missing. These kind of indentation issues are not acceptable. I am seeing frequently. 
data, field,callback

Reply:
indentation is done and pushed